src/lib/sort/sort-header.html

```diff
@@ -7,8 +7,7 @@
 
   <div *ngIf="_isSorted()"
        class="mat-sort-header-arrow"
-       [class.mat-sort-header-asc]="_sort.direction == 'asc'"
-       [class.mat-sort-header-desc]="_sort.direction == 'desc'">
+       [@indicatorRotate]="this._sort.direction">
```

Comment:
Although the use of `this` in template is valid (until now), it may be broken in future `angular` versions, since it's not even documented. Btw, I could not find the question I had seen in the SO that discussed this better.